core/src/main/scala/filodb.core/store/ChunkSetInfo.scala

```diff
@@ -274,4 +274,51 @@ class FilteredChunkInfoIterator(base: ChunkInfoIterator, filter: ChunkSetInfo =>
     gotNext = false   // reet so we can look for the next item where filter == true
     nextnext
   }
+}
+
+/**
+ * A RowReader that returns info about each ChunkSetInfo set via setInfo with a schema like this:
+ * 0:  ID (Long)
+ * 1:  NumRows (Int)
+ * 2:  startTime (Long)
+ * 3:  endTime (Long)
+ * 4:  numBytes(Int) of chunk
+ * 5:  readerclass of chunk
+ */
+class ChunkInfoRowReader(column: Column) extends RowReader {
```

Comment:
Following additional information will be useful:
* encoding format for the chunk
* whether it is a write-buffer

Eventually, these can be added:
* Block information (odp or regular) 
* Lock status (how many readers) for debugging locking issues.

Reply:
Yes definitely.   BTW the first two, encoding format for the chunk is what the "readerclass" is.  If the class is one of the uncompressed ones then it's a write buffer (there isn't really any other current way to tell, other than it being the first chunk).   

In future versions we can add a function which can derive the writebuffer-ness from the existing info...

agreed on block info